SPEC.md

```diff
@@ -0,0 +1,97 @@
+# Introduction
+
+The purpose of this index is to provide an overview of the testing approaches to be implemented by Ziggurat. It is intended to evolve as the framework matures, leaving room for novel cases and extensions of existing cases, as called for by any protocol idiosyncrasies that may come to light during the development process.
+
+Some test cases have been consolidated when similar behaviour is tested with differing messages. The final implementation of these cases will be subject to factors such as node setup and teardown details, test run time (and potentially runtime) constraints, readability and maintainability.
+
+## Special Considerations
+
+_To be done_
```

Comment:
I'd suggest removing this section entirely until there is something to document here.

Reply:
Done